### ISSUE TITLE
unix.execve() bash/zsh APE fails with: ape error: ./zsh: APE phdrs must be 4096-aligned and 4096-padded

### ISSUE THREAD

**Issue by OEvgeny** [jart/cosmopolitan#1095]
I noticed that at least from redbean's Lua interpreter I'm not able to run bash/zsh APEs obtained from cosmos: https://cosmo.zip/pub/cosmos/bin/

Running like this:

```Lua
  -- myscript.lua
  if assert(unix.fork()) == 0 then
    local zsh = unix.commandv('./zsh')
    unix.execve(zsh, { 'zsh', '--help' })
  else
    assert(unix.wait())
  end
```

```
./redbean -i myscript.lua
```

Getting the following error:
```
ape error: ./zsh: APE phdrs must be 4096-aligned and 4096-padded
```

I'm not experiencing any issue running make or unzip with this method. The only problematic ones so far are: `bash` and `zsh`.

I checked binaries and they seem to work well on my system

**Comment by jart** [comment]
It's possible you have an old version of ape loader somewhere. Could you try this for me?

```
ape/apeuninstall.sh
ape/apeinstall.sh
```

**Comment by OEvgeny** [comment]
This is a clean OS I haven't used APE before. After doing the reinstall process from above comment `execve()` works correctly.

I also verified if I run apeuninstall.sh the issue appears again. I can run other APEs, just these ones fail when running through `execve()`. Think I never had ape loader installed system wide and didn't have any issues running any ape binaries except this one.

I'm running NixOS:
```
Linux 6.6.10 #1-NixOS SMP PREEMPT_DYNAMIC Fri Jan  5 14:19:45 UTC 2024 x86_64 GNU/Linux
```

Closing this as it has the way around. I'm happy to help testing further.

Thank you for this great project!
